pkg/cmd/pr/checkout/checkout.go

```diff
@@ -240,6 +240,24 @@ func localBranchExists(client *git.Client, b string) bool {
 	return err == nil
 }
 
+// Warn if the pull request and the remote branch have diverged.
+func warnIfDiverged(opts *CheckoutOptions, pr *api.PullRequest) {
+	if len(pr.Commits.Nodes) == 0 {
+		return
+	}
```

Comment:
**How might the PR not have commits?**

I ask out of genuine curiosity of the user workflow involved.